54_spiral_matrix.py

```diff
@@ -0,0 +1,63 @@
+from typing import List
+from enum import Enum
+
+
+class Solution:
+
+    def spiralOrder(self, matrix: List[List[int]]) -> List[int]:
+
+        """
+            // Time Complexity : O(mn)
+                    'm' is the number of rows, 'n' is the number of columns
+            // Space Complexity : O(mn)
+            // Did this code successfully run on Leetcode :
```

Comment:
Recheck space here

Reply:
my bad. should it be **O(1)**  I always think the output list is also considered. I keep making this mistake. Will be careful nex time